testsuite/features/secondary/srv_salt_git_pillar.feature

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@skip_if_github_validation
+@sle_minion
+@scope_salt
+Feature: Salt Master integration with Git Pillar
+
+  Scenario: Preparing Git Pillar configuration for Salt Master
+    When I setup a git_pillar environment on the Salt Master
+    And I wait for "30" seconds
+    Then file "/etc/salt/master.d/zz-testing-gitpillar.conf" should exist on server
+
+  Scenario: Check for the expected pillar data after enabling Git Pillar
+    When I refresh the pillar data
+    Then the pillar data for "git_pillar_foobar" should be "12345" on "sle_minion"
```

Comment:
I understand it's only an example, but could we use something that makes a bit more of human sense?

Reply:
Hmmm, I want that this pillar data injected by Salt Git Pillar has a meaningful name for debugging in case of issues, that is why I chose `git_pillar_foobar`.

Reply:
I was referring to the 12345 part

Reply:
What is wrong with `12345` as a value for this test pillar data?

IMO the `12345` makes total sense to me as a value for a pillar data :smile:

Reply:
Agreed with Eric that value `12345` is also acceptable here in this context.